Sources/Tracing/Docs.docc/Guides/ImplementATracer.md

```diff
@@ -0,0 +1,186 @@
+# Implement a Tracer
+
+## Overview
+
+This guide is aimed at ``TracerProtocol`` and ``InstrumentProtocol`` implementation authors.
```

Comment:
IIRC we renamed them to `Tracer` and `Instrument`?

Reply:
Yes sorry I should have been explicit that not all code is cleanedup here yet